### PR TITLE
Update 22/23 Team

### PR DESCRIPTION
**Pull request by ruioliveira02**

### CODE REVIEW

**Review by miguelbrandao — APPROVED** [review]
:tada:

File: public/team.json [hunk 1]
```diff
@@ -3,185 +3,223 @@
...
-            "image": "team/filipe-felicio.jpg"
+            "image": "team/goncalo-rodrigues.jpg"
           }
         },
         {
-          "Carlos Ferreira": {
+          "Vitor Leite": {
             "position": "Tesoureiro",
-            "image": "team/carlos-ferreira.jpg"
+            "image": "team/vitor-leite.jpg"
```
Comment (miguelbrandao) [inline public/team.json:26]: `vitor-lelis.jpg`

File: public/team.json [hunk 2]
```diff
@@ -3,185 +3,223 @@
...
-          "Filipe Felício": {
+          "Gonçalo Rodrigues": {
             "position": "Vice-Presidente",
-            "image": "team/filipe-felicio.jpg"
+            "image": "team/goncalo-rodrigues.jpg"
           }
         },
         {
-          "Carlos Ferreira": {
+          "Vitor Leite": {
```
Comment (miguelbrandao) [inline public/team.json:24]: Vitor Lelis?

File: public/team.json [hunk 3]
```diff
@@ -3,185 +3,223 @@
...
-            "position": "Codiretora",
-            "image": "team/matilde-bravo.jpg"
+          "Rui Lopes": {
+            "position": "Codiretor",
+            "image": "team/rui-lopes.jpg"
           }
         },
         {
-          "Rui Oliveira": {
+                "Tiago Pereira" : {
```
Comment (miguelbrandao) [inline public/team.json:52]: Indentation

File: public/team.json [hunk 4]
```diff
@@ -3,185 +3,223 @@
...
             "position": "Codiretor",
-            "image": "team/rui-oliveira.jpg"
+            "image": "team/tiago-pereira.jpg"
+                }
+        },
+        {
+          "Enzo Vieira": {
+                  "position": "Codiretor",
```
Comment (miguelbrandao) [inline public/team.json:59]: Indentation

File: public/team.json [hunk 5]
```diff
@@ -3,185 +3,223 @@
...
+      }
+    }
+  ],
   "Mesa da Assembleia Geral": [
     {
-      "Miguel Brandão": {
+      "Tiago Soares": {
         "position": "Presidente",
-        "image": "team/miguel-brandao.jpg"
+        "image": "team/tiago-soares.jpg"
```
Comment (miguelbrandao) [inline public/team.json:190]: Missing picture

**Review by miguelbrandao — CHANGES_REQUESTED** [review]
Pass the `team.json` file through a prettier to fix the indentation issues (there are several)

File: public/team.json (hunk 2 above)
Comment (ruioliveira02) [reply public/team.json:24]: His last name is Leite. It is this way in the current site as well

**Review by miguelbrandao — DISMISSED** [review]
🎉

File: public/team.json [hunk 6]
```diff
@@ -3,186 +3,226 @@
     {
       "Presidência": [
         {
-          "Nelson Estevão": {
-            "position": "Presidente",
-            "image": "team/nelson-estevao.jpg"
+          "Sofia Gomes": {
+            "position": "Presidene",
```
Comment (nelsonmestevao) [inline public/team.json:7]:
```suggestion
            "position": "Presidente",
```

**Review by nelsonmestevao — CHANGES_REQUESTED** [review]

File: public/team.json (hunk 6 above)
Comment (miguelbrandao) [reply public/team.json]: Mmmmm I don't know. I think I prefer the way it used to be

**Review by nelsonmestevao — APPROVED** [review]

**Review by miguelbrandao — APPROVED** [review]

File: public/team.json (hunk 6 above)
Comment (nelsonmestevao) [reply public/team.json]: It was just a suggestion 😅

**Pull request merged**